client/src/components/organisms/AgendaSection.tsx

```diff
@@ -1,15 +1,30 @@
 import React from "react";
 import { Box, Card } from "@/components/atoms";
 import { SectionHeader } from "@/components/molecules/SectionHeader";
+import { OngoingAgendaCard } from "../molecules/OngoingAgendaCard";
+import { Agenda, OngoingAgenda } from "biseo-interface/agenda";
+import { useAgenda } from "@/services/agenda";
+
+const isOngoingAgenda = (agenda: Agenda): agenda is OngoingAgenda => {
+  return agenda.status === "ongoing";
+};
 
 export const AgendaSection: React.FC = () => {
+  const { ongoingAgendas } = useAgenda(state => ({
+    ongoingAgendas: state.agendas.filter(isOngoingAgenda),
+  }));
+
+  const ongolingAgendaCards = ongoingAgendas.map((agenda, id) => (
+    <OngoingAgendaCard key={id} agenda={agenda}></OngoingAgendaCard>
+  ));
```

Comment:
이 부분에서 
`map((agenda, id) => ...`로 받아오는 경우 `id`에는 해당 아이템의 index가 들어가게 되는데, index를 key로 쓰는 것은 보통 권장되지 않습니다.

자세한 내용은 [여기](https://medium.com/sjk5766/react-%EB%B0%B0%EC%97%B4%EC%9D%98-index%EB%A5%BC-key%EB%A1%9C-%EC%93%B0%EB%A9%B4-%EC%95%88%EB%90%98%EB%8A%94-%EC%9D%B4%EC%9C%A0-3ce48b3a18fb)를 참고해 보면 될 것 같아요:

이 경우에는 `agenda`에 `id`가 존재하니 `agenda.id`를 키로 사용하면 될 것 같아요.
```suggestion
  const ongolingAgendaCards = ongoingAgendas.map(agenda => (
    <OngoingAgendaCard key={agenda.id} agenda={agenda}></OngoingAgendaCard>
  ));
```